ephios/api/templates/api/access_token_list.html

```diff
@@ -0,0 +1,98 @@
+{% extends "core/settings/settings_base.html" %}
+{% load rich_text %}
+
+{% load i18n %}
+{% block settings_content %}
+    <div class="block-center">
+        <h1>{% translate "Your API Tokens" %}</h1>
+        <p>
+            {% blocktranslate trimmed %}
+                API Tokens let other applications authenticate at ephios.
+                They might be created manually or when logging in at a third party sites.
+                If you are unsure about the origins of a token or the security of the third party application
+                please revoke the token.
+            {% endblocktranslate %}
+        </p>
+
+        <a href="{% url "api:settings-access-token-create" %}"
+           class="btn btn-secondary mb-3">{% translate "Create new API token" %}</a>
+
+        {% if not request.GET.show_inactive %}
+            <a href="?show_inactive=1" class="btn btn-secondary ms-1 mb-3">{% translate "Show inactive" %}</a>
```

Comment:
do you think that this will be helpful for users? I think that it may be too much